.goreleaser.yml

```diff
@@ -0,0 +1,5 @@
+# yaml-language-server: $schema=https://goreleaser.com/static/schema-pro.json
```

Comment:
This is going to cause some issues because that remote config expects a `schema.graphql` file as part of the build. Take a look at `load-balancer-operator` for the local config we use for non graphql builds.